utils/cliutils/commandsflags.go

```diff
@@ -1032,7 +1032,7 @@ var flagsMap = map[string]cli.Flag{
 	},
 	xrayScan: cli.StringFlag{
 		Name:  xrayScan,
-		Usage: "[Default: false]  Set to true if you'd like artifacts upload only after passing successfully an Xray scan.` `",
+		Usage: "[Default: false] Set to true if you'd like artifacts upload only after passing successfully an Xray scan.` `",
```

Comment:
Set to true if you'd like artifacts upload only after passing successfully an Xray scan.
-->
Set if you'd like all files to be scanned by Xray on the local file system prior to the upload, and skip the upload if any of the files are found vulnerable.